### PR TITLE
Update layout docs to reflect changes made to the template

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
- Fixed small typos across docs.
- Added ‘Use X layout when…’ and ‘Don’t use X layout when…’ sections.
- Added introductory text for each example.
- Renamed section including API information and props to ‘Configuring X layout’.
- Added ‘Help and Support’ section to each page.
- Amended headings to ensure we use sentence case throughout.

### CODE REVIEW

**Review by sion-docs — APPROVED** [review]

File: packages/core/stories/help-and-support.doc.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Help and support
+
+Please check the issues section of the github repository for any development or design related questions or issues you have.
```
Comment (joshwooding) [inline packages/core/stories/help-and-support.doc.mdx:3]:
```suggestion
Please check the issues section of the GitHub repository for any development or design related questions or issues you have.
```

File: packages/core/stories/layout/flex-layout.doc.stories.mdx [hunk 2]
```diff
@@ -83,20 +68,41 @@ You can use the default breakpoints or pass in your own.
...
+
+### API
+
+```
+import { FlexLayout, FlexItem } from "@jpmorganchase/uitk-core";
+```
+
+### Props
+
+**ResponsiveProp** refers to a prop than take either multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes.
```
Comment (joshwooding) [inline packages/core/stories/layout/flex-layout.doc.stories.mdx:97]:
```suggestion
**ResponsiveProp** refers to a prop that takes either multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes.
```
?

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by sion-docs — DISMISSED** [review]

**Review by Fercas123 — DISMISSED** [review]

File: packages/core/stories/layout/flex-layout.doc.stories.mdx [hunk 3]
```diff
@@ -30,51 +31,35 @@ For layouts where you need to control the two dimensions, see [Grid layout](?pat
...
+You want your layout to have a constant direction. In this case we recommend you having a look at [Stack layout](?path=/docs/documentation-core-layout-stacklayout--page) or [Flow layout](?path=/docs/documentation-core-layout-flowlayout--page).
 
-```
-import { FlexLayout, FlexItem } from "@jpmorganchase/uitk-core";
-```
-
-## Props
+### Examples
 
-**ResponsiveProp** refers to a prop than take either multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes.
```
Comment (Fercas123) [inline packages/core/stories/layout/flex-layout.doc.stories.mdx:48]: should be ` that takes`

File: packages/core/stories/layout/grid-layout.doc.stories.mdx [hunk 4]
```diff
@@ -25,76 +26,87 @@ For more complex layouts, it supports spanning across columns and rows.
...
 
-```
-import { GridLayout, GridItem } from "@jpmorganchase/uitk-core";
-```
+You want your layout to flow in a single dimension. In this case we recommend you having a look at [Stack layout](?path=/docs/documentation-core-layout-stacklayout--page), [Flow layout](?path=/docs/documentation-core-layout-flowlayout--page) or [Flex layout](?path=/docs/documentation-core-layout-flexlayout--page).
 
-## Props
+### Examples
 
-**ResponsiveProp** refers to a prop than take either multiple values (one per breakpoint) or a single value that would take effect across all different screen sizes.
```
Comment (Fercas123) [inline packages/core/stories/layout/grid-layout.doc.stories.mdx:44]: I think this is happening in more places, might be worth checking all the example's text

**Comment by JoanaMMoreira** [comment]
> sorry, just saw this from Josh's comments, there is also the lowecase github one but i couldn't find it

I think that was the only place we had it as far as I can tell

**Review by sion-docs — DISMISSED** [review]

File: packages/core/stories/layout/border-layout.doc.stories.mdx [hunk 5]
```diff
@@ -24,19 +25,17 @@ The border layout is built using [Grid layout](?path=/docs/documentation-core-la
...
-```
-import { BorderLayout, BorderItem } from "@jpmorganchase/uitk-core";
-```
+You want a fixed layout with a flexible main region.
 
-## Props
+### Don't use Border layout when...
 
-<ArgsTable of={BorderLayout} />
+You want a customised layout that you can control in both row and column dimensions. In this case we recommend you having a look at [Grid layout](?path=/docs/documentation-core-layout-gridlayout--page).
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.doc.stories.mdx:34]:
```suggestion
You want a customized layout that you can control in both row and column dimensions. In this case we recommend you having a look at [Grid layout](?path=/docs/documentation-core-layout-gridlayout--page).
```

File: packages/core/stories/layout/grid-layout.doc.stories.mdx [hunk 6]
```diff
@@ -25,76 +26,87 @@ For more complex layouts, it supports spanning across columns and rows.
...
-<ArgsTable of={GridLayout} />
+You want your layout to flow in a single dimension. In this case we recommend you having a look at [Stack layout](?path=/docs/documentation-core-layout-stacklayout--page), [Flow layout](?path=/docs/documentation-core-layout-flowlayout--page) or [Flex layout](?path=/docs/documentation-core-layout-flexlayout--page).
 
-## Examples
+### Examples
 
-### Grid Item inside default grid layout
+#### Grid Item inside default grid layout
 
 A grid item allows more complex controls to the default behaviour of a grid child, like defining how many rows or columns will it span trough and it's alignment
```
Comment (joshwooding) [inline packages/core/stories/layout/grid-layout.doc.stories.mdx:46]:
```suggestion
A grid item allows more complex controls to the default behavior of a grid child, like defining how many rows or columns will it span trough and it's alignment
```

File: packages/core/stories/layout/grid-layout.doc.stories.mdx [hunk 7]
```diff
@@ -25,76 +26,87 @@ For more complex layouts, it supports spanning across columns and rows.
...
 You have an item that needs to take the space of more than one row/column or when you have an item that requires a different alignment to the component's default.
 
 <Canvas>
   <Story id="core-layout-gridlayout-griditem--grid-item-wrapper" />
 </Canvas>
 
-### Responsive
+#### Responsive
 
 The grid supports the use of responsive props, so you can define different behaviours based on viewport size.
```
Comment (joshwooding) [inline packages/core/stories/layout/grid-layout.doc.stories.mdx:58]:
```suggestion
The grid supports the use of responsive props, so you can define different behaviors based on viewport size.
```

File: packages/core/stories/layout/grid-layout.doc.stories.mdx [hunk 8]
```diff
@@ -25,76 +26,87 @@ For more complex layouts, it supports spanning across columns and rows.
...
 All grid props are responsive, and their values can be individually customised per breakpoint.
```
Comment (joshwooding) [inline packages/core/stories/layout/grid-layout.doc.stories.mdx:59]:
```suggestion
All grid props are responsive, and their values can be individually customized per breakpoint.  
```

File: packages/core/stories/layout/layer-layout.doc.stories.mdx [hunk 9]
```diff
@@ -25,54 +26,68 @@ It can be used to build dialog or drawer components and it can be positioned in
...
+Instead you want your content to be split between pages or tabs. In this case we recommend you having a look at [Deck layout](?path=/docs/documentation-core-layout-decklayout--page).
 
-## Props
+### Examples
 
-<ArgsTable of={LayerLayout} />
+#### Layer positioning
 
-## Examples
+The component will be displayed in the center position by default but you can customise this via the `position` prop as per the following examples.
```
Comment (joshwooding) [inline packages/core/stories/layout/layer-layout.doc.stories.mdx:45]:
```suggestion
The component will be displayed in the center position by default but you can customize this via the `position` prop as per the following examples.
```

File: packages/core/stories/layout/layer-layout.doc.stories.mdx [hunk 10]
```diff
@@ -25,54 +26,68 @@ It can be used to build dialog or drawer components and it can be positioned in
...
+##### Bottom positioned
 
 <Canvas>
   <Story id="core-layout-layerlayout--layer-layout-bottom" />
 </Canvas>
 
-### Reduced motion
+#### Reduced motion
+
+You can customise or disable the animations using the `prefers-reduced-motion` CSS media feature.
```
Comment (joshwooding) [inline packages/core/stories/layout/layer-layout.doc.stories.mdx:73]:
```suggestion
You can customize or disable the animations using the `prefers-reduced-motion` CSS media feature.
```

File: packages/core/stories/layout/parent-child-layout.doc.stories.mdx [hunk 11]
```diff
@@ -24,38 +25,52 @@ The parent and child layout is built using [Flex layout](?path=/docs/documentati
...
 The parent and child component changes to a stacked view once it approaches a specified breakpoint that can be customised via the `stackedAtBreakpoint` prop.
 
 <Canvas>
   <Story id="core-layout-parentchildlayout--toolkit-parent-child-layout-stacked" />
 </Canvas>
 
-### Reduced motion
+#### Reduced motion
+
+You can customise or disable the animations using the `prefers-reduced-motion` CSS media feature.
```
Comment (joshwooding) [inline packages/core/stories/layout/parent-child-layout.doc.stories.mdx:48]:
```suggestion
You can customize or disable the animations using the `prefers-reduced-motion` CSS media feature.
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by Fercas123 — APPROVED** [review]

**Review by sion-docs — APPROVED** [review]

**Comment by sion-docs** [comment]
I have noticed some inconsistency around sentence style in the headings, the following need to be updated:
- Border layout
- Flow layout
- ParentChild layout
- Stack layout

**Pull request merged**